proto/src/oauth2.rs

```diff
@@ -236,6 +250,8 @@ pub struct OAuth2RFC9068TokenExtensions {
 pub struct AccessTokenResponse {
     pub access_token: String,
     pub token_type: AccessTokenType,
+    /// RFC8693 issued_token_type for the representation of the issued token.
+    pub issued_token_type: Option<String>,
```

Comment:
Should be an enum based on the variants in https://www.rfc-editor.org/rfc/rfc8693#TokenTypeIdentifiers